code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1106,6 +1116,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 /obj/item/pda/Destroy()
 	GLOB.PDAs -= src
+	UnregisterSignal(src, COMSIG_LIGHT_EATER_ACT)
```

Comment:
No need to unregister source signals on destroy, that's done automatically already.

Reply:
It's the standard for components and the like.

Reply:
Check the definition of `Destroy()` at the base.
This is already handled by default.

Reply:
Done